stl/inc/algorithm

```diff
@@ -9228,8 +9289,29 @@ namespace ranges {
 #endif // __cpp_lib_concepts
 #endif // _HAS_CXX17
 
+template <class _FwdIt, class _Pr>
+_INLINE_VAR constexpr bool _Is_min_max_optimization_safe =
+    _Iterator_is_contiguous<_FwdIt>&&
+        conjunction_v<disjunction<is_integral<_Iter_value_t<_FwdIt>>, is_pointer<_Iter_value_t<_FwdIt>>>,

```

Comment:
Do we really need the short circuit from `conjunction` or should we use the more efficient variable templates (applies only for clang as MSVC optimizes this internally)

Reply:
I don't know

Reply:
In general, I think that `conjunction` is preferable (when it is reasonably readable), since the short-circuiting can sometimes be beneficial. If Clang really cares about throughput, they should implement the same kind of optimizations :smile_cat: